src/renderer.hpp

```diff
@@ -68,6 +68,13 @@ namespace wr
 		virtual void Resize(std::uint32_t width, std::uint32_t height) = 0;
 		
 		std::optional<Window*> m_window;
+		
+		struct EffectsEnabled
```

Comment:
The render system shouldn't be in charge of this. The user should be.

Reply:
Deprecated now that FG rebuilding is scrapped. Should not have been in the code base anymore, my bad! Should be fixed